src/spicy-vodka-pasta.md

```diff
@@ -0,0 +1,39 @@
+# Spicy Vodka Pasta
+
+A tasty and flavorful pasta.
+
+- ⏲️ Prep time: 10 min
+- 🍳Cook time: 20 min
+
+## Ingredients (Makes 2)
+
+- 1/4 cup olive oil
+- 1 clove of garlic, diced
+- 1/4 of a green onion
+- 1/4 cup tomato paste
+- 1/4 cup heavy cream
+- 1 tablespoon vodka (alternatively, to taste)
+- 1 1/2 teaspoon red pepper flakes
+- 225g preferred pasta
```

Comment:
```suggestion
- 8 oz (225g) preferred pasta
```
You are using cups for everything else, it is kind of weird to suddenly shift to S.I. units